tests/base.py

```diff
@@ -196,6 +196,10 @@ def assertStatus(self, response, code):
         """
         code = str(code)
         msg = 'Response status was %s, not %s.' % (response.output_status, code)
+
+        if response.json:
+            msg += ' Response body was %s.' % json.dumps(response.json)
```

Comment:
I am happy to make it easier for those that have trouble reading unformatted JSON :smirk_cat: 
Do you think we need a newline before printing the JSON in that case? Something like:
'''
' Response body was:\n%s.' % json.dumps(response.json, sort_keys=True, indent=4, separators=(',', ': '))
'''